### PR TITLE
Add blog post: LibrePCB Talk at FOSDEM 2018

### PR DESCRIPTION
**Pull request by ubruhin**

### CODE REVIEW

File: content/blog/2018-01-02_librepcb_at_fosdem_18.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+date: 2018-01-02
+title: LibrePCB Talk at FOSDEM 2018
+author: U. Bruhin
+---
+
+The [*Free and Open Source Software Developers' European Meeting (FOSDEM)*](https://fosdem.org/)
+is a great event for developers and users interested in free software. It is
+hosted at the ULB Solbosch Campus in Brussels, Belgium. This year it takes
+place at **February 3&4**.
```
Comment (dbrgn) [inline content/blog/2018-01-02_librepcb_at_fosdem_18.md]: I think it's `on`, not `at`. Also, I'd replace the `&` with a `/`.

File: content/blog/2018-01-02_librepcb_at_fosdem_18.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+[![FOSDEM](/img/fosdem_2018_banner.png)](https://fosdem.org/)
+
+There are many different tracks available, but for us the most interesting one
+is the [**CAD and Open Hardware devroom**](https://fosdem.org/2018/schedule/track/cad_and_open_hardware/).
+It contains talks about PCB design, FPGA degin, SPICE simulation and open
```
Comment (dbrgn) [inline content/blog/2018-01-02_librepcb_at_fosdem_18.md]: `s/degin/design`

File: content/blog/2018-01-02_librepcb_at_fosdem_18.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+hardware in general.
+
+And this year we have submitted the [**Introduction to LibrePCB**](https://fosdem.org/2018/schedule/event/cad_librepcb/)
+talk to present our project! The talk takes place at **Saturday, February 3,
```
Comment (dbrgn) [inline content/blog/2018-01-02_librepcb_at_fosdem_18.md]: Again, `at` -> `on`

**Comment by ubruhin** [comment]
@dbrgn Thanks for reviewing!

Regarding the `February 3 / 4` term: Unfortunately it seems to be impossible to write `3/4` (no spaces) because it is automatically converted to ¾ :sob: According [these docs](https://gohugo.io/getting-started/configuration/) it's possible to disable this conversion, but:
> Caveat: Even with fractions = false, Blackfriday still converts 1/2, 1/4, and 3/4 respectively to ½ (&frac12;), ¼ (&frac14;) and ¾ (&frac34;), but only these three.

:sob::sob::sob:

**Comment by dbrgn** [comment]
https://getpelican.com 😛

**Comment by ubruhin** [comment]
> https://getpelican.com :stuck_out_tongue:

Adding spaces to `3 / 4` is easier and solves the problem too :grin: 

Ready to merge now?

**Pull request merged**